chromadb/utils/embedding_functions.py

```diff
@@ -1027,18 +1027,27 @@ class FastEmbedEmbeddingFunction(EmbeddingFunction[Documents]):
     def __init__(
         self,
         model_name: str = "BAAI/bge-small-en-v1.5",
+        batch_size: int = 256,
```

Comment:
Is there any technical limitation (aka upper bound) to this? I guess the same question applies to threads.

For threads, is there a default value that fastembed uses (e.g., num cores or even 2x num cores)? Should we add a note to guide users on how to select the "appropriate" number of threads?

Reply:
The default values that I've set are the same as the FastEmbed defaults.

There isn't a documented bound or recommended value. It's usually figured out by trial and error on specific systems.

Reply:
Fair enough.